python/lsst/rubintv/production/processingControl.py

```diff
@@ -280,15 +285,25 @@ class PipelineComponents:
     steps: list[str]
     pipelineFile: str
 
-    def __init__(self, registry: Registry, pipelineFile: str, steps: list[str]) -> None:
+    def __init__(
+        self,
+        registry: Registry,
+        pipelineFile: str,
+        steps: list[str],
+        overrides: list[tuple[str, str, str]] | None = None,
+    ) -> None:
         self.uris: dict[str, str] = {}
         self.graphs: dict[str, PipelineGraph] = {}
         self.graphBytes: dict[str, bytes] = {}
         self.pipelineFile = pipelineFile
 
         for step in steps:
             self.uris[step] = pipelineFile + f"#{step}"
-            self.graphs[step] = Pipeline.fromFile(self.uris[step]).to_graph(registry=registry)
+            pipeline = Pipeline.fromFile(self.uris[step])
+            if overrides:
+                for override in overrides:
+                    pipeline.addConfigOverride(*override)
+            self.graphs[step] = pipeline.to_graph(registry=registry)
```

Comment:
I think this isn't going to work when the number of steps is greater than one, because any given override is going to be for a particular task, and that task is only going to appear in one step.  I think you're going to need to start checking `override[0] in pipeline.task_labels`.

Reply:
Ah, good point, thanks! (I hope to barely use this feature, because I really want to run them as-is, but found this was necessary for changing the output for the cp_verify pipelines, but it's certainly a good point in general and not a bad thing to add).